appveyor.yml

```diff
@@ -2,7 +2,8 @@ image: Visual Studio 2017
 init:
   - git config --global core.autocrlf input
 build_script:
-  - cmd: build.cmd NuGet /p:BuildWithStyleCop=yes
```

Comment:
I personally would squash your last three commits into one. Otherwise I don't see any issues.

Reply:
Seems appveyor didn't exec build process right, as it take 5 mins to build vs travis 20 min